package.json

```diff
@@ -13,6 +13,7 @@
   "dependencies": {
     "algo-msgpack-with-bigint": "^2.1.0",
     "buffer": "^6.0.2",
+    "core-js": "^3.9.1",
```

Comment:
This is a poly-fill so that new JS features work in all browsers. See [this](https://www.valentinog.com/blog/preset-env/#enter-core-js) for more info.